packages/themes/src/aspect.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  aspectRatio: 9 / 16,
```

Comment:
Common ratios are 16:9, 4:3, 21:9.
9/16 is understandable, however it is kind of backwards.